src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -97,17 +99,10 @@ public async Task MigrateAsync(BoundSonarQubeProject oldBinding, IProgress<Migra
             logger.WriteLine(MigrationStrings.Finished);
         }
 
-        private LegacySettings GetLegacySettings()
-        {
-            // TODO - calculate the partial paths to the ruleset and SonarLint.xml files
-            // for both C# and VB.NET - #4362 and #4363
-            return new LegacySettings("folder", "cs ruleset", "cs xml", "vb ruleset", "vb xml");
-        }
-
-        private Task<string> GetFileContentAsync(string filePath)
+        private System.Threading.Tasks.Task<string> GetFileContentAsync(string filePath)
```

Comment:
I perhaps didn't need to qualify this `Task<>` and the one in line 105; the VS2019 build failure might only be complaining about the non-generic `Task` type being ambiguous. However, it won't do any harm to be explicit.